upup/pkg/fi/cloudup/gce/network.go

```diff
@@ -162,17 +162,17 @@ func performNetworkAssignmentsIPAliases(ctx context.Context, c *kops.Cluster, cl
 	// CIDRs should be in the RFC1918 range, but otherwise we have no constraints
 	networkCIDR := "10.0.0.0/8"
 
-	podCIDR, err := used.Allocate(networkCIDR, net.CIDRMask(14, 32))
```

Comment:
The changes in this file supports clusters with: 
- 8k nodes
- 65k services

https://cloud.google.com/kubernetes-engine/docs/concepts/alias-ips#defaults_limits

We should create a cilium-gce network provider to resolve https://github.com/kubernetes/kops/issues/16071 and avoid creating an unreasonable number of routes per node.